docker/docker-compose.yml

```diff
@@ -15,14 +15,17 @@ services:
       - 9973:9973/tcp
       - 9973:9973/udp
 # ws and http port might not be widely exposed, but to your internal clique/apps/network only.
-# Configure this with caution
-#      - 11973:11973
-#      - 12973:12973
+# Defaults are set to not open these two ports publicly. Configure this with caution.
+      - 127.0.0.1:11973:11973
+      - 127.0.0.1:12974:12973
```

Comment:
arf, sorry, my port 12973 was used by another instance ;) Fixing this one.
As you mentioned , host networking is usually not recommended (because not kubernetes friendly, which is the next step here ;)), so better avoiding.